resources/js/common/components/UserAvatar/UserAvatar.tsx

```diff
@@ -25,7 +26,7 @@ export const UserAvatar: FC<UserAvatarProps> = ({
 }) => {
   const { cardTooltipProps } = useCardTooltip({ dynamicType: 'user', dynamicId: displayName });
 
-  const canLinkToUser = displayName && !deletedAt;
+  const canLinkToUser = displayName && !deletedAt && !isGone;
```

Comment:
Moderators can view the profile page of a banned user. It might be nice to leave the link for them? I'm not sure how often (if at all) it would be used.

Reply:
Messed around with this for about an hour. This is tricky for two reasons:
* `isGone` doesn't differentiate between deleted and banned (intentionally)
* `canManageUsers` would be a new global prop which needs to be read by `<UserAvatar />`

Not impossible by any means, but complex enough that I'm wondering if anyone cites this as being inconvenient. Going to punt to being a possible future task based on if we get feedback.